### PR TITLE
add ext module project example

### PR DESCRIPTION
**Pull request by Camork**

### CODE REVIEW

File: gradle/ext/login-web-ext/build.gradle [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apply plugin: 'com.liferay.osgi.ext.plugin'
+
+dependencies {
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
```
Comment (gamerson) [inline gradle/ext/login-web-ext/build.gradle]: We can't use `default` version.  I don't even know what that does.

File: liferay-workspace/ext/login-web-ext/gradlew [hunk 2]
```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```
Comment (gamerson) [inline liferay-workspace/ext/login-web-ext/gradlew]: We don't need to add gradle wrapper to workspace projects.

**Review by gamerson — CHANGES_REQUESTED** [review]

File: gradle/ext/login-web-ext/build.gradle (hunk 1 above)
Comment (Camork) [reply gradle/ext/login-web-ext/build.gradle]: changed to 3.0.0

File: liferay-workspace/ext/login-web-ext/gradlew (hunk 2 above)
Comment (Camork) [reply liferay-workspace/ext/login-web-ext/gradlew]: removed

**Comment by Camork** [comment]
resend [here](https://github.com/gamerson/liferay-blade-samples/pull/529)

**Pull request closed without merging**